docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -1,23 +1,183 @@
+
+```yaml
+components.*.certificate.keystore.file: <SAF-key-ring-from-AT-TLS-rule>
+components.*.certificate.keystore.type: JCERACFKS
+components.*.certificate.keystore.password: password
+components.*.certificate.keystore.alias: <certificate alias / label from AT-TLS rule>
 ```
+
+Finally, if there is an outbound AT-TLS rule configured for the link between the API Gateway and z/OSMF, update or set the `zowe.zOSMF.scheme` to `http`.
+
 :::note
-This procedure does not configure AT-TLS on z/OS, but rather enables API ML to work with AT-TLS in place.
+AT-TLS is not yet supported in the API Cloud Gateway Mediation Layer component.
 :::
+
+## Security considerations
+
+Configuring AT-TLS for the Zowe API Mediation Layer needs careful consideration regarding the security settings, especially around the Client Certificate authentication feature in the Zowe API Mediation Layer components as well as for the Onboarded services that support the x.509 client certificates authentication scheme.
+
+In general terms, the outbound AT-TLS rules (i.e. to make a transparent https call through http) that are configured to send the server certificate should be limited to the services that __require__ service to service authentication, such as the case of the API Gateway authenticating with the Discovery Service.
+
+**Note:** The Discovery Service endpoints are not reachable by standard API Gateway routing by default.
+
+## Inbound and Outbout AT-TLS rules
+
+This section describes the suggested AT-TLS settings. It is meant as a guideline and helps showcase important aspects.
+
+### Inbound rules
+
+A generic inbound rule can be set for all of the core API Mediation Layer services:
+
+```pagent
+TTLSRule ApimlServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 10310-10320
+  Jobname ZWE*
+  Direction Inbound
+  TTLSGroupActionRef ServerGroupAction
+  TTLSEnvironmentActionRef ApimlServerEnvironmentAction
+  TTLSConnectionActionRef ApimlServerConnectionAction
+}
+
+TTLSGroupAction ServerGroupAction
+{
+  TTLSEnabled On
+}
+
+TTLSEnvironmentAction ApimlServerEnvironmentAction
+{
+  HandshakeRole ServerWithClientAuth
+  EnvironmentUserInstance 0
+  TTLSEnvironmentAdvancedParmsRef ServerEnvironmentAdvParms
+  TTLSKeyringParmsRef ApimlKeyring
+}
+
+TTLSConnectionAction ApimlServerConnectionAction
+{
+  HandshakeRole ServerWithClientAuth
+  TTLSCipherParmsRef CipherParms
+  TTLSConnectionAdvancedParmsRef ApimlConnectionAdvParms
+}
+```
+
+Note the setting `HandshakeRole` as this is meant for the core services which authenticate through certificates with each other and will allow the API Gateway to receive and accept X.509 client certificates from API Clients.
+
+### Outbound rules
+
+- For z/OSMF
+
+```pagent
+TTLSRule ApimlZosmfClientRule
+{
+  LocalAddr All
+  LocalPortRange 1024-65535
+  RemoteAddr All
+  RemotePortRange 449
+  Jobname ZWEAAG*
+  Direction Outbound
+  TTLSGroupActionRef ClientGroupAction
```

Comment:
The cipher's rule is not mentioned. Please add it with the link like https://www.ibm.com/docs/en/i/7.5?topic=settings-cipher-suite-configuration

Reply:
they don't want IBM links in documentation, they tend to be difficult to maintain, I can comment that they can get the list of supported ciphers and the matching between.
Also this link goes to an "IBM i" product